src/components/charts/bar-chart.tsx

```diff
@@ -15,7 +15,7 @@ interface IBarProps {
 
 const defaultOptions: ChartOptions = {
   title: {
-    display: true,
+    display: false,
```

Comment:
I assume we no longer want to show the title since the panel header should act as a title.